src/GafferImage/Display.cpp

```diff
@@ -474,6 +477,9 @@ void Display::setDriver( IECoreImage::DisplayDriverPtr driver, bool copy )
 	setupDriver( copy ? new GafferDisplayDriver( *gafferDisplayDriver ) : gafferDisplayDriver );
 
 	driverCountPlug()->setValue( driverCountPlug()->getValue() + 1 );
+
+	// The copied driver will have it's tile channelDataCounts reset, so we need to force a refresh
```

Comment:
And perhaps something like this to match?

```suggestion
	// The copied tiles will have `cacheForDataCount == 0`, so we set `channelDataCountPlug` to 1
	// to force a refresh on the next compute.
```